test/docker/compose.go

```diff
@@ -558,7 +558,6 @@ func (d *Compose) startCluster(ctx context.Context, size int, settings map[strin
 		config2["CLUSTER_DATA_BIND_PORT"] = "7103"
 		config2["CLUSTER_JOIN"] = fmt.Sprintf("%s:7100", Weaviate1)
 		eg.Go(func() (err error) {
-			time.Sleep(time.Second * 3)
```

Comment:
Due to the peculiarities of member-list bootstrapping in a cluster, t his delay may be necessary to prevent acceptance test failures

Reply:
I don't mind putting it back, but wouldn't it make more sense instead to have a healthcheck for "memberlist" accept ready ? Maybe we can make it a quick endpoint to add.

Reply:
Some delay is necessary here because the joining nodes depend on the first node to bootstrap the cluster. A better approach would be to handle this in Weaviate by having the followers wait for the first node.